packages/rocketchat-theme/client/imports/base.css

```diff
@@ -4745,13 +4745,13 @@ a + br.only-after-a {
 }
 
 .embedded-view {
-	& .flex-tab-bar {
-		display: none;
-	}
-
 	& .messages-container {
 		border-width: 0;
 
+		& .flex-tab-container {
```

Comment:
@rodrigok @sampaiodiego it's correct, idk why but I saw a space after the dot yesterday.

@gdelavald actually you need to put `&` before. We are using [this css nesting draft](http://tabatkins.github.io/specs/css-nesting) on our implementation